src/App.tsx

```diff
@@ -1,61 +1,156 @@
+import { ChangeEvent, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('');
+  const [updatedTodos, setUpdatedTodos] = useState(todos);
+  const [noUser, setNoUser] = useState(false);
+  const [noTitle, setNoTitle] = useState(false);
```

Comment:
Rename [boolean variable](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)